charts/gitlab-ci-pipelines-exporter/values.yaml

```diff
@@ -181,3 +183,7 @@ ingress:
   tls:
     - hosts:
         - gcpe.example.com
+
+rbac:
+  enabled: false
+  clusterRole: cluster-admin
```

Comment:
What's the purpose of using `cluster-admin` ? 🤔

Reply:
It can be any clusterRole existed on your  Kubernetes. I will change it to empty so that everyone can modify it based on their needs

Reply:
sounds great thanks!